### ISSUE TITLE
response in progress audio cue loops forever sometimes

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#194424]
1. with screen reader mode on, in the inline chat, type `generate some tests for this`, with a selection
2. 🐛 in progress response audio cue plays forever until I reload the window even though the response has come back

**Comment by meganrogge** [comment]
There's still a case that's not covered.

**Comment by meganrogge** [comment]
Example for what caused this:

1. Enable screen reader mode
2. Open vscode, run "@workspace which parts of the code have the most to do comments"
3. It will take a long time to reply. Ensure the chat in progress audio cue stops playing.

**Comment by jrieken** [comment]
@meganrogge I have reverted your changes because `release` does _not_ mean accept session. Release happens on accept, error, cancel etc. Understand this as disposing an object

**Comment by meganrogge** [comment]
Yep @jrieken in that case, we want to accept the response, which cancels the audio cue but doesn't announce the response was completed. Pls revert the revert so this is fixed
